cms/static/cms/sass/components/pagetree/_tree.scss

```diff
@@ -46,6 +46,36 @@
     .cms-hover-tooltip {
         z-index: auto;
     }
+    .sr-only {
```

Comment:
shouldn't we add this as a helper class in general for the cms? like cms-sr-only?